recipes/snmf/meta.yaml

```diff
@@ -0,0 +1,26 @@
+about:
+  home: https://github.com/avengerpb/snmf_conda
+  license: Artistic-2.0
+  summary: Estimating individual admixture coefficients based on sparse non-negative matrix factorization and population genetics
+
+build:
+  number: 0
+  skip: False
+
+requirements:
+  build:
+    - perl-bioperl
+    - perl-lib
+
+package:
+  name: snmf
+  version: 1.0.0
+
+source:
+   fn: 1.5.tar.gz
+   url: https://github.com/avengerpb/snmf_conda/archive/1.5.tar.gz
```

Comment:
A cleaner way to do this would be to not use your separate repo and instead have `snmf.sh` and `Snmf.pl` as extra files included in this recipe that are copied over to `$PREFIX/bin` (e.g., in `build.sh` you would have `cp $RECIPE_DIR/snmf.sh $PREFIX/bin/snmf.sh`).

I also just noticed you are copying a pre-compiled `plink` over, which is unlikely to pass tests on  travis-ci. So your recipe would need to include run dependencies of `plink` (or `plink2`), `perl-bioperl`, and `perl-switch`.

It does not look like we have a recipe for `perl-switch`, so you can create one and include it in this PR.